app/assets/stylesheets/main.css.scss

```diff
@@ -1925,6 +1925,27 @@ figure.fluidratio {
 }
 
 /*
+  Branding
+*/
+.branding-container {
+  clear:  both;
+
+  hr {
```

Comment:
I'd prefer a css class here, in this case using tag name seems very unlikely to cause issues but I'd like to stick to the principle of not using them except in resets.